examples/mobile/ScheduleApp/js/app.js

```diff
@@ -185,7 +187,20 @@
 			</div>
 		</div>`
 
-			container.innerHTML += HTMLTemplate;
+			switch (type) {
+				case "on":
+					container.innerHTML += HTMLTemplateOn;
+					break;
+				case "off":
```

Comment:
case "off":
case "onoff":

Reply:
Our eslint rules don't allow to duplicate case labels. This is defined in `eslint:recommended` which we follow in our ruleset.